src/lib/utils/shuffle.ts

```diff
@@ -22,11 +22,19 @@ const rearrangeLettersWithRng = (letters: string[]) => {
 	return newLetters;
 };
 
-export const rearrangeLettersManually = (letters: string[]) => [
-	letters[2],
-	letters[0],
-	letters[3],
-	letters[1]
-];
+export const rearrangeLettersManually = (letters: string[]) => {
+	// split the array into two parts
+	const firstHalf = letters.slice(0, letters.length / 2);
+	const secondHalf = letters.slice(letters.length / 2);
+
+	// alternate the letters from the two parts
+	const rearrangedLetters = [];
+	for (let i = 0; i < letters.length; i++) {
+		if (i % 2 === 0) rearrangedLetters.push(firstHalf[i / 2]);
+		else rearrangedLetters.push(secondHalf[(i - 1) / 2]);
+	}
+
+	return rearrangedLetters;
+};
```

Comment:
this is cool, if you tested this out then it's probably fine, but dereferencing arrays always makes me worried of out of bounds exceptions. so I might make a few unit tests for this lad later today

Reply:
Sounds good! We can resolve this once the unit tests are up.